paymentsheet/src/main/java/com/stripe/android/customersheet/injection/CustomerSheetViewModelModule.kt

```diff
@@ -95,4 +96,11 @@ internal class CustomerSheetViewModelModule {
             )
         )
     }
+
+    @Provides
+    fun originalPaymentSelection(): PaymentSelection? = originalPaymentSelection
+
+    private companion object {
+        private val originalPaymentSelection: PaymentSelection? = null
+    }
```

Comment:
So is this always going to be `null`?

Reply:
Only when it is injected in the customer sheet. It will be set when we retrieve it from the adapter on intiailize.